### PR TITLE
multi stop point ui 

### PR DESCRIPTION
**Pull request by OmerGery**

### CODE REVIEW

File: examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/AddressSelector/SearchBar.js [hunk 1]
```diff
@@ -167,22 +211,53 @@ const SearchBar = ({
...
+            }
           }}
         />
       </Row>
+
+
     );
-  });
+  };
+  const buildSps = () => (
```
Comment (ariel-weiss) [inline examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/AddressSelector/SearchBar.js:261]: imo it's better to take this and export to a different file as a separate component (the next step would be to extract all multi-sp logic into a custom hook to separate between render & logic), to make this more manageable

File: examples/client/Locomotion/package.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
     "react-native-config": "^1.4.5",
     "react-native-date-picker": "^4.2.6",
     "react-native-device-info": "^8.7.1",
+    "react-native-draglist": "^3.3.0",
```
Comment (ariel-weiss) [inline examples/client/Locomotion/package.json]: 🤩

File: examples/client/Locomotion/src/lib/ride/utils.ts [hunk 3]
```diff
@@ -16,15 +20,28 @@ export const formatSps = function (stopPoints: any) {
...
   const v = n % 100;
   return n + (s[(v - 20) % 10] || s[v] || s[0]);
 };
 
+export const formatSpText = (sp: any) => {
+  const uiSpType = formatUiDisplaySpType(sp, null);
+  const { ordinalDesc } = sp;
+  const isMultiSp = isMulti(sp);
+  const isOnlyMultiSp = isMultiSp && sp.amountOfSpsFromType === 2;
+  const numberPart = `${sp.ordinalDesc && !isOnlyMultiSp ? `${getOrdinal(isMultiSp ? ordinalDesc : ordinalDesc + 1)} ` : ''}`;
```
Comment (ariel-weiss) [inline examples/client/Locomotion/src/lib/ride/utils.ts]: what is numberPart ? can you change it to more readable name (or add comment)? 🙏  essentially I don't understand the output of formatSpText method

**Review by ariel-weiss — APPROVED** [review]

File: examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/AddressSelector/SearchBar.js (hunk 1 above)
Comment (OmerGery) [reply examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/AddressSelector/SearchBar.js:261]: agreed :) , will adjust for next multi sp ui phase , a lot of content already for this PR

**Pull request merged**